db/migrate/20170207021033_add_phone_number_to_club_application.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhoneNumberToClubApplication < ActiveRecord::Migration
+  def change
+    add_column :club_applications, :phone_number, :string
```

Comment:
@zachlatta: `string` is used for all of the other non long form data types in the database. I did it to be consistent.